src/routes/views/app/GreetingView.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { render, screen } from '@testing-library/react'
+import { Greeting } from '../../../entities/Greeting'
+import { mockGreetingAPI } from '../../../networking/api'
+import GreetingView from './GreetingView'
+
+/*
+ * This is an example of a view test.
+ * As the view calls the hook which calls the api, the api has to be mocked. See greeting-hook test.
+ * A view test should check if the correct api calls are made, if the components are at the right place
+ * and if the empty state of the view (if no data is available) is correctly displayed.
+ */
+
+const defaultMock = () => ({
+    get: jest.fn(async (name: string) => ({ id: 0, content: 'Hello test-user' })),
+    save: jest.fn(async (greeting: Greeting) => greeting),
+})
+
+test('GreetingView shows greeting', async () => {
+    const mock = mockGreetingAPI(defaultMock())
+    render(<GreetingView />)
+
+    expect(mock.get).toBeCalledTimes(1)
+
+    await screen.findByText('Hello test-user')
+})
+
+// const notFoundMock = () => ({
+//     get: jest.fn(async (name: string) => Promise.reject()),
+//     save: jest.fn(async (greeting: Greeting) => greeting),
+// })
+
+// test('GreetingView shows empty greeting', async () => {
+//     const mock = mockGreetingAPI(notFoundMock())
+//     render(<GreetingView />)
+
+//     expect(mock.get).toBeCalledTimes(1)
+
+//     await screen.findByText('Nobody is here.')
+// })
```

Comment:
Is this still needed? Or why is it commented out? Maybe add a comment in code on why it is commented out.